.github/workflows/integration-test.yml

```diff
@@ -8,11 +8,22 @@ on:
 
 jobs:
 
-  build:
-
+  integration-tests:
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
Strategy allows you to run the same job with with different setups. In this case we run the integration test with local data(as defined in the repo) and remote data

Reply:
Fixed it, we'll skip the run if a remote URL is not available